flask_appbuilder/models/sqla/__init__.py

```diff
@@ -30,8 +30,7 @@ class SQLA(SQLAlchemy):
 
         Use it and configure it just like flask_SQLAlchemy
     """
-    def make_declarative_base(self, metadata=None):
-        """Creates the declarative base."""
+    def make_declarative_base(self, model, metadata=None):
```

Comment:
Can you explain this change?

Reply:
This is the current signature of `make_declarative` base as of flask-sqlachemy 2.3: https://github.com/mitsuhiko/flask-sqlalchemy/blob/master/flask_sqlalchemy/__init__.py#L734